doc/source/whatsnew/v1.3.0.rst

```diff
@@ -693,10 +686,10 @@ Deprecations
 - Deprecated passing arguments as positional in :meth:`DataFrame.set_index` (other than ``"keys"``) (:issue:`41485`)
 - Deprecated passing arguments as positional (except for ``"levels"``) in :meth:`MultiIndex.set_levels` (:issue:`41485`)
 - Deprecated passing arguments as positional in :meth:`DataFrame.sort_index` and :meth:`Series.sort_index` (:issue:`41485`)
-- Deprecated passing arguments as positional in :meth:`DataFrame.drop_duplicates` (except for ``subset``), :meth:`Series.drop_duplicates`, :meth:`Index.drop_duplicates` and :meth:`MultiIndex.drop_duplicates`(:issue:`41485`)
+- Deprecated passing arguments as positional in :meth:`DataFrame.drop_duplicates` (except for ``subset``), :meth:`Series.drop_duplicates`, :meth:`Index.drop_duplicates` and :meth:`MultiIndex.drop_duplicates` (:issue:`41485`)
```

Comment:
Not necessarily for this PR, but there are enough deprecations i think it might be worth subsections for e.g. positional arguments (or at least collect them all together)

Reply:
@MarcoGorelli is planning to tidy that up. https://github.com/pandas-dev/pandas/issues/41485#issuecomment-854714232

Reply:
yup - those deprecations are all done now so I'll do this week